server/internal/api/user_statistics.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2023 OpenIM open source community. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+
+	"github.com/openimsdk/openkf/server/internal/common"
+	"github.com/openimsdk/openkf/server/internal/common/response"
+	requestparams "github.com/openimsdk/openkf/server/internal/params/request"
+	responseparams "github.com/openimsdk/openkf/server/internal/params/response"
+	"github.com/openimsdk/openkf/server/internal/service"
+	"github.com/openimsdk/openkf/server/internal/utils"
+)
+
+// GetUserStatistics
+// @Tags user
+// @Summary GetUserStatistics
+// @Description get user statistics
+// @Security  ApiKeyAuth
+// @Produce application/json
+// @Param data body requestparams.GetUsersStatisticsParams true "GetUsersStatisticsParams"
+// @Success 200 {object}  response.Response{msg=string} "Success"
+// @Router /api/v1/user/statistics [post].
+func GetUserStatistics(c *gin.Context) {
+	uuid, err := utils.GetUserUUID(c)
+	if err != nil {
+		response.FailWithCode(common.UNAUTHORIZED, c)
+
+		return
+	}
+
+	var params requestparams.GetUsersStatisticsParams
+	err = c.ShouldBindJSON(&params)
+	if err != nil {
+		response.FailWithCode(common.INVALID_PARAMS, c)
```

Comment:
Use log printing?

Reply:
I will update `FailWithCode ` in other PR with log embeded.